modules/thermal_hydraulics/test/tests/components/supersonic_inlet/tests

```
@@ -0,0 +1,12 @@
+[Tests]
+  [not_implemented]
+    type = 'RunException'
+    input = 'err.i'
+    expect_err = "This component is not implemented for single-phase flow."
+    max_parallel = 1
+    ad_indexing_type = 'global'
+    design = 'SupersonicInlet.md'
+    requirement = 'The system shall throw an error if the SupersonicInlet component is used.'
```

"The system shall report an error..."